packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -277,7 +277,7 @@ class _ModalBottomSheetState<T> extends State<_ModalBottomSheet<T>> {
               child: CustomSingleChildLayout(
                 delegate: _ModalBottomSheetLayout(animationValue, widget.isScrollControlled),
                 child: BottomSheet(
-                  color: widget.route.bottomSheetColor,
+                  color: widget.route.backgorundColor,
```

Comment:
Nit-picking at this point, but typo for backgroundColor throughout this commit

Reply:
Augh!  Will fix.  Thanks for catching that.